components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -5,20 +5,15 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-import {Parser, ProcessNodeDefinitions} from 'html-to-react';
+import {Posts} from 'mattermost-redux/constants';
+import {getChannelsNameMapInCurrentTeam} from 'mattermost-redux/selectors/entities/channels';
 
 import store from 'stores/redux_store.jsx';
 
 import * as PostUtils from 'utils/post_utils.jsx';
 import * as TextFormatting from 'utils/text_formatting.jsx';
 import * as Utils from 'utils/utils.jsx';
 
```

Comment:
This import arrangement made the `post_type.test.jsx` fail. Import should be arranged like:
```
import store from 'stores/redux_store.jsx';

import * as PostUtils from 'utils/post_utils.jsx';
import * as TextFormatting from 'utils/text_formatting.jsx';
import * as Utils from 'utils/utils.jsx';

import {Posts} from 'mattermost-redux/constants';   // eslint-disable-line import/order
import {getChannelsNameMapInCurrentTeam} from 'mattermost-redux/selectors/entities/channels';   // eslint-disable-line import/order
```

Reply:
Also, please do update snapshot of `tests/plugins/post_type.test.jsx` and see to it that it passed the tests.